<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_Indexes</class>
 <widget class="QWidget" name="Prefs_Indexes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Indexes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="indexListBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QPushButton" name="indexAddButton">
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="shortcut">
          <string>Alt+A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="indexDeleteButton">
         <property name="text">
          <string>&amp;Delete</string>
         </property>
         <property name="shortcut">
          <string>Alt+D</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>666</width>
        <height>300</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Properties:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="tocDestFrameLabel">
                 <property name="text">
                  <string>Destination Frame:</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="itemDestFrameComboBox">
                 <property name="toolTip">
                  <string>The frame the table of contents will be placed into</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="listNonPrintingCheckBox">
               <property name="toolTip">
                <string>Include frames that are set to not print</string>
               </property>
               <property name="text">
                <string>List Non-Printing Entries</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="combinePageNumbersCheckBox">
               <property name="toolTip">
                <string>Combine entries when a word is repeated on the same page</string>
               </property>
               <property name="text">
                <string>Combine Page Numbers</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="combineIdenticalEntriesCheckBox">
               <property name="toolTip">
                <string>Combine entries for the same word</string>
               </property>
               <property name="text">
                <string>Combine Identical Entries</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="caseSensitiveCombinationCheckBox">
               <property name="toolTip">
                <string>When comparing identical entries, compare with case sensitivity</string>
               </property>
               <property name="text">
                <string>Case Sensitive</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="autoCapitalizeEntriesCheckBox">
               <property name="toolTip">
                <string>Autocapitalize the index entry</string>
               </property>
               <property name="text">
                <string>AutoCapitalize Entries</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="addAlphaSeparatorsCheckBox">
               <property name="toolTip">
                <string>Add alphabetical separators in the index</string>
               </property>
               <property name="text">
                <string>Add Alphabetical Separators</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Styles:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="styleHeadingLabel">
                 <property name="text">
                  <string>Heading:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="styleHeadingComboBox"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="styleSeparatorsLabel">
                 <property name="text">
                  <string>Separators:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="styleSeparatorComboBox"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="styleLevel1Label">
                 <property name="text">
                  <string>Level 1:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="styleLevel1ComboBox"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="styleLevel2Label">
                 <property name="text">
                  <string>Level 2:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="styleLevel2ComboBox"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="styleLevel3Label">
                 <property name="text">
                  <string>Level 3:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="styleLevel3ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
